testing/src/test/java/org/jdbi/v3/testing/junit5/JdbiSqliteExtendWithTest.java

```diff
@@ -36,4 +36,14 @@ public void testHandleIsAlive(Handle handle) {
 
         assertThat(one).isOne();
     }
+
+    @Test
+    public void dataSourceSharesData(Jdbi jdbi) {
+        jdbi.useHandle(h -> {
```

Comment:
what is this testing? I am not sure what that change does and this test has no assertions.

Reply:
The test currently (before this change) fails - as the original reporter posts, every time you `useHandle` right now you get a *different* database, which is nearly useless.

Reply:
I added an `assertNoExceptionThrownBy` to make this more explicit